djangocms_references/helpers.py

```diff
@@ -92,6 +158,12 @@ def contenttype_values_queryset(queryset):
 
 
 def convert_plugin_querysets_to_sources(querysets):
+    """Convert provided plugin querysets to a form containing
```

Comment:
I found this a bit confusing, when I read "form" first time I thought you meant a django form class (which would have been weird!).